### PR TITLE
2097 - Update some personalization colors

### LINKED ISSUE #2097: Homepages: Text Color is too dark on the hero
**Describe the bug**
Noticed the text color isnt changing correctly on home page her widgets: http://master-enterprise.demo.design.infor.com/components/homepage/example-hero-widget.html

**To Reproduce**
1. Go to http://master-enterprise.demo.design.infor.com/components/homepage/example-hero-widget.html
2. Examine the text in the hero area.

**Expected behavior**
Text is legible and changes based on the personalization colors. Can possible use the new text color classes http://master-enterprise.demo.design.infor.com/components/personalize/example-classes.html

**Version**
- ids-enterprise: v4.18.0

### PR DESCRIPTION
**Pull request by deep7102**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**
There was Failed QA because some text not readable, so this PR will update some of the personalization colors.

**Related github/jira issue (required)**:
Fixes #2097

**Steps necessary to review your pull request (required)**:
- Pull this branch and build/run the demo app
- Navigate to http://localhost:4000/components/homepage/example-hero-widget.html
- Click top right `...` action button to change theme/colors
- Inspect the header/tootip/chart colors should be easy readable

**Included in this Pull Request**:
- [x] A note to the change log.

### CODE REVIEW

File: src/components/bar/bar.js [hunk 1]
```diff
@@ -444,6 +445,9 @@ Bar.prototype = {
           const y = isTooltipBottom ? yPosS : (yPosS - size.height - 13);
 
           if (content !== '') {
+            if (charts.tooltip && charts.tooltip.length) {
+              charts.tooltip[isPersonalizable ? 'addClass' : 'removeClass']('isPersonalizable');
```
Comment (tmcconechy) [inline src/components/bar/bar.js]: Class should be `is-personalizable` for consistency

File: src/components/bullet/bullet.js [hunk 2]
```diff
@@ -177,6 +179,9 @@ Bullet.prototype = {
             const y = rect.top - size.height + $(window).scrollTop() - 5;
 
             if (content !== '') {
+              if (charts.tooltip && charts.tooltip.length) {
+                charts.tooltip[isPersonalizable ? 'addClass' : 'removeClass']('isPersonalizable');
```
Comment (tmcconechy) [inline src/components/bullet/bullet.js]: Class name `is-personalizable` for consistency

File: src/components/charts/charts.js [hunk 3]
```diff
@@ -90,7 +90,9 @@ charts.appendTooltip = function appendTooltip(extraClass) {
  * @returns {void}
  */
 charts.hideTooltip = function hideTooltip() {
-  d3.select('#svg-tooltip').classed('is-hidden', true).style('left', '-999px');
+  d3.select('#svg-tooltip')
+    .classed('isPersonalizable', false)
```
Comment (tmcconechy) [inline src/components/charts/charts.js]: Class name `is-personalizable` for consistency

File: src/components/column/column.js [hunk 4]
```diff
@@ -703,6 +704,9 @@ Column.prototype = {
           }
 
           if (content !== '') {
+            if (charts.tooltip && charts.tooltip.length) {
+              charts.tooltip[isPersonalizable ? 'addClass' : 'removeClass']('isPersonalizable');
```
Comment (tmcconechy) [inline src/components/column/column.js]: Class name `is-personalizable` for consistency

File: src/components/line/line.js [hunk 5]
```diff
@@ -479,6 +480,9 @@ Line.prototype = {
             ((rect.left + (rect.width / 2)) - (size.width / 2)) : posX;
 
           if (content !== '') {
+            if (charts.tooltip && charts.tooltip.length) {
+              charts.tooltip[isPersonalizable ? 'addClass' : 'removeClass']('isPersonalizable');
```
Comment (tmcconechy) [inline src/components/line/line.js]: Class name `is-personalizable` for consistency

File: src/components/pie/pie.js [hunk 6]
```diff
@@ -420,6 +421,10 @@ Pie.prototype = {
           let y = offset.top;
           const padding = 5;
 
+          if (charts.tooltip && charts.tooltip.length && (isTop || isRight || isBottom || isLeft)) {
```
Comment (tmcconechy) [inline src/components/pie/pie.js]: Class name `is-personalizable` for consistency

File: src/components/radar/radar.js [hunk 7]
```diff
@@ -442,6 +443,9 @@ Radar.prototype = {
         }
 
         tooltipInterval = setTimeout(() => {
+          if (charts.tooltip && charts.tooltip.length) {
```
Comment (tmcconechy) [inline src/components/radar/radar.js]: Class name `is-personalizable` for consistency

File: src/components/sparkline/sparkline.js [hunk 8]
```diff
@@ -144,6 +145,9 @@ Sparkline.prototype = {
             const posY = rect.top - size.height - 5; // 5 is extra padding
 
             if (content !== '') {
+              if (charts.tooltip && charts.tooltip.length) {
```
Comment (tmcconechy) [inline src/components/sparkline/sparkline.js]: Class name `is-personalizable` for consistency

File: src/components/sparkline/sparkline.js [hunk 9]
```diff
@@ -229,6 +233,9 @@ Sparkline.prototype = {
           const posY = rect.top - size.height - 8;
 
           if (content !== '') {
+            if (charts.tooltip && charts.tooltip.length) {
```
Comment (tmcconechy) [inline src/components/sparkline/sparkline.js]: Class name `is-personalizable` for consistency

File: src/components/tooltip/tooltip.js [hunk 10]
```diff
@@ -629,6 +629,10 @@ Tooltip.prototype = {
 
     const mouseUpEventName = this.isTouch ? 'touchend' : 'mouseup';
 
+    // Personalizable the toolbar
+    const isPersonalizable = self.element.closest('.is-personalizable').length > 0;
+    self.tooltip[0].classList[isPersonalizable ? 'add' : 'remove']('isPersonalizable');
```
Comment (tmcconechy) [inline src/components/tooltip/tooltip.js]: Class name `is-personalizable` for consistency

File: src/components/tooltip/tooltip.js [hunk 11]
```diff
@@ -812,6 +816,7 @@ Tooltip.prototype = {
       return;
     }
 
+    this.tooltip[0].classList.remove('isPersonalizable');
```
Comment (tmcconechy) [inline src/components/tooltip/tooltip.js]: Class name `is-personalizable` for consistency

File: src/components/personalize/personalize.styles.js [hunk 12]
```diff
@@ -435,6 +435,112 @@ function personalizeStyles(colors) {
...
+.header.is-personalizable .toolbar [class^='btn']:focus:not(.hide-focus),
+.header.is-personalizable .flex-toolbar [class^='btn']:focus:not(.hide-focus),
+.subheader.is-personalizable .toolbar [class^='btn']:focus:not(.hide-focus),
+.subheader.is-personalizable .flex-toolbar [class^='btn']:focus:not(.hide-focus) {
+  box-shadow: 0 0 0 2px transparent,
+    0 0 0 1px ${colors.subtext},
+    0 0 2px 1px ${colors.subtext};
+}
+
+.tooltip.isPersonalizable {
```
Comment (EdwardCoyle) [inline src/components/personalize/personalize.styles.js]: needs to be `.is-personalizable`

**Comment by deep7102** [comment]
@tmcconechy @EdwardCoyle @davidcarlsonberg 
Updated now, the css class name as kebob case `is-personalizable`

File: src/components/sparkline/sparkline.js [hunk 13]
```diff
@@ -229,6 +233,9 @@ Sparkline.prototype = {
...
+              charts.tooltip[isPersonalizable ? 'addClass' : 'removeClass']('isPersonalizable');
```
Comment (EdwardCoyle) [inline src/components/sparkline/sparkline.js]:
```suggestion
              charts.tooltip[isPersonalizable ? 'addClass' : 'removeClass']('is-personalizable');
```

**Pull request merged**